python/robot/sample_kobiton_app.robot

```diff
@@ -4,6 +4,7 @@ Library           AppiumLibrary
 *** Variables ***
 ${REMOTE_URL}           username:apiKey
 ${PLATFORM_NAME}        Android
+${PLATFORM_VERSION}     8*
```

Comment:
@idxn Please don't specify the platformVersion, we want to make sure the scripts can run when having an online device

Reply:
Ok. I'll remove it out.